README.md

```diff
@@ -1,8 +1,8 @@
 [![Test Workflow](https://github.com/opensearch-project/alerting/workflows/Test%20Workflow/badge.svg)](https://github.com/opensearch-project/alerting/actions)
 [![codecov](https://codecov.io/gh/opensearch-project/alerting/branch/main/graph/badge.svg)](https://codecov.io/gh/opensearch-project/alerting)
 [![Documentation](https://img.shields.io/badge/api-reference-blue.svg)](https://opensearch.org/docs/latest/monitoring-plugins/alerting/api/)
-[![Chat](https://img.shields.io/badge/chat-on%20forums-blue)](https://discuss.opendistrocommunity.dev/c/alerting/)
-![PRs welcome!](https://img.shields.io/badge/PRs-welcome!-success)
+[![Chat](https://img.shields.io/badge/chat-on%20forums-blue)](https://forum.opensearch.org/c/plugins/alerting/7)
+[![PRs welcome!](https://img.shields.io/badge/PRs-welcome!-success)]
```

Comment:
The `]` at the end of line 5 should be removed. It caused this slight issue with the formatting of the README.
![Screen Shot 2022-09-07 at 9 46 50 AM](https://user-images.githubusercontent.com/79280347/188935044-73e107e0-1d65-42d5-9975-7e8d03837e5e.png)